client/packages/sdk/src/circuit/tx.ts

```diff
@@ -32,6 +24,51 @@ export class Tx {
     this.exportMode = exportMode;
   }
 
+  /**
+   *
+   * @param tx - The transaction to send
+   * @param options - Options for transaction
+   *
+   * @returns Promise
+   */
+
+  async signAndSend(tx: SubmittableExtrinsic, options: Partial<SignerOptions>): Promise<string> {
+    let exportObj: ExtrinsicExport;
+
+    if (this.exportMode) {
+      exportObj = new ExtrinsicExport(tx, this.signer.address);
+    }
+
+    return new Promise((resolve, reject) =>
```

Comment:
Let's use `async-await` syntax, to conform to standards.
Also, it makes the code more readable.